src/transformers/pipelines/token_classification.py

```diff
@@ -60,6 +66,9 @@ class AggregationStrategy(ExplicitEnum):
         grouped_entities (`bool`, *optional*, defaults to `False`):
             DEPRECATED, use `aggregation_strategy` instead. Whether or not to group the tokens corresponding to the
             same entity together in the predictions or not.
+        stride (`int`, *optional*, defaults to `None`):
```

Comment:
```suggestion
        stride (`int`, *optional*):
```
We don't write the default if it's None as a convention in the repo.